pytravharv/TravHarvConfigBuilder.py

```diff
@@ -385,19 +390,18 @@ def _makeTravHarvConfigPartFromDict(
 
         return TravHarvConfig(travharvconfig)
 
-    def _check_snooze(self, snooze_time, lastmodified_admin, name_config):
-        if name_config in lastmodified_admin:
-            lastmodified_admin_time = lastmodified_admin[name_config]
-            log.debug(
-                "lastmodified_admin_time: {}".format(lastmodified_admin_time)
+    def _check_snooze(self, snooze_time, name_config):
+        log.debug(
+            "Checking if config {} is older then {} minutes".format(
+                name_config, snooze_time
             )
-            tosnooze_time = lastmodified_admin_time + timedelta(
-                minutes=snooze_time
+        )
+        log.debug(graph_name_to_uri(name_config))
+        return (
+            False
+            if not self._rdf_store_access
+            else self._rdf_store_access.lastmod_for_context(
+                graph_name_to_uri(name_config)
             )
-
-            log.debug("tosnooze_time: {}".format(tosnooze_time))
-
-            if tosnooze_time > datetime.now():
-                return True
-            else:
-                return False
+            < datetime.now() - timedelta(minutes=snooze_time)
```

Comment:
rdf-store has built in check for last mod checking?

https://github.com/vliz-be-opsci/py-RDF-store/blob/main/pyrdfstore/store.py#L56

also: it was recently fixed to handle possible timezone issues

Reply:
issue mentioned in #16